test_runner/regress/test_logical_replication.py

```diff
@@ -578,7 +578,10 @@ def test_subscriber_synchronous_commit(neon_simple_env: NeonEnv, vanilla_pg):
     vanilla_pg.safe_psql("create extension neon;")
 
     env.create_branch("subscriber")
-    sub = env.endpoints.create("subscriber")
+    # We want all data to fit into shared_buffers because later we stop
+    # safekeeper and insert more; this shouldn't cause page requests as they
+    # will be stuck.
+    sub = env.endpoints.create("subscriber", config_lines=["shared_buffers=128MB"])
```

Comment:
We are trying to enable LFC for tests by default.
That's why we changed shared buffers to 512kB and faced this problem.
We can leave `shared_buffers` value at 1 MB or make it slightly larger, e.g., 2MB, and the problem will go away.
But there is a question: why doesn't it cache the data to LFC if there is not enough `shared_buffers` memory?